AutomatedLab/AutomatedLabAzure.psm1

```diff
@@ -1393,14 +1394,14 @@ function Get-LabAzureLabSourcesContentRecursive
     $temporaryContent = $StorageContext | Get-AzStorageFile
     foreach ($item in $temporaryContent)
     {
-        if ($item.GetType().FullName -eq 'Microsoft.Azure.Storage.File.CloudFileDirectory')
+        if ($item.CloudFileDirectory -is [Microsoft.Azure.Storage.File.CloudFileDirectory])
```

Comment:
```suggestion
        if ($item.CloudFileDirectory)
```

The type check seems redundant now. I assume CloudFileDirectory will always be of this type